docs/coordinates/spectralcoord.rst

```diff
@@ -0,0 +1,530 @@
+(in this case with the radio convention)::
+
+    >>> sc_feat_rest.to(u.km / u.s, doppler_convention='radio', doppler_rest=115.27120180 * u.GHz)  # doctest: +REMOTE_DATA
+    <SpectralCoord [-11.90160347, -27.50828539, -30.10939904] km / s
+        observer:
+          <ICRS Coordinate: (x, y, z) in m
+              (-1.25867767e+11, -7.48979688e+10, -3.24757657e+10)
+           (v_x, v_y, v_z) in km / s
+              (9.34149908, 20.49579745, 7.99178839)>
+        target:
+          <ICRS Coordinate: (ra, dec, distance) in (deg, deg, pc)
+              (65.497625, 19.53511111, 144.321)
+           (pm_ra_cosdec, pm_dec, radial_velocity) in (mas / yr, mas / yr, km / s)
+              (1.37949782e-15, 1.46375638e-15, 23.9)>
+        observer to target (computed from above):
+          radial_velocity=0.0 km / s
+          redshift=0.0
+    	doppler_rest=115.2712018 GHz
+    	doppler_convention=radio>
+
+Note that these resulting velocities are different from the ``radial_velocity``
+property (which is still zero here) - the latter is the difference in velocity
+between observer and target, while the former are how much the spectral values
+are Doppler shifted by relative to the rest frequency or wavelength.
+
+So if the features are indeed from 12CO, they have velocities of approximately -11.9, -27.5 and
+-30.1 km/s relative to the T tau rest frame.
+
+.. _spectralcoord-common-frames:
+
+Common velocity frames
+======================
+
+Any valid astropy coordinate frame can be passed to the
+:meth:`~astropy.coordinates.SpectralCoord.with_observer_velocity`
+method, including string aliases such as ``icrs``. Below we list some of the
+frames commonly used to define spectral coordinates in:
+
+The velocity frames available as constants on the |SpectralCoord| class are:
+
+========================== =================================================
+Frame name                 Description
+========================== =================================================
+``'gcrs'``                 Geocentric frame (defined as stationary relative to the GCRS origin)
+``'icrs'``                 Barycentric frame (defined as stationary relative to the ICRS origin)
+``'hcrs'``                 Heliocentric frame (defined as stationary relative to the HCRS origin)
+``'lsrk``                  Kinematic Local Standard of Rest (LSRK),
+                           defined as having a velocity of 20 km/s towards
+                           18h +30d (B1900) relative to the Solar System
+                           Barycenter [1]_.
+``'lsrd'``                 Dynamical Local Standard of Rest (LSRD),
+                           defined as having a velocity of U=9 km/s,
+                           V=12 km/s, and W=7 km/s in Galactic coordinates
+                           (equivalent to 16.552945 km/s towards l=53.13
+                           and b=25.02) [2]_.
+``'lsr'``                  A more recent definition of the Local Standard
```

Comment:
If this is more recent, shouldn't it be used in the examples above?

Reply:
Yes and no - it's more recent but also not necessarily 'standard' yet - LSRD and LSRK have been widely used for years. I actually am not sure it was a good idea to have a plain LSR paper and just base it off a recent paper, because it's by no means a standard, just one study.

Reply:
OK, makes sense. Maybe an idea to even discuss this when you introduce `LSRD`.

Reply:
I reworded this